experiments/testing_archive/_cpid_unique.py

```diff
@@ -15,7 +15,7 @@
 class TestExperiment(ExperimentPrototype):
 
     def __init__(self):
-        cpid = 151 # This is not a unique number 
+        cpid = 151  # This is not a unique number 
```

Comment:
did you mean to change the name of this file?

Reply:
Yes, we can't have 'test' in the filename, as the code that I added to check for *local* CPID uniqueness doesn't look at files that start with 'test'. I think I made a comment about this in the csv file.